VaticanBAV/et/BAVet131.xml

```diff
@@ -27,52 +27,1023 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            Pss 42-44 (for the rain, <locus from="17r" to="18r"/>), Pss 45-47 (for the fruits, <locus from="18r" to="19r"/>), Pss 48-50 (for the water of the rivers, 
+                                            <locus from="19r" to="20r"/>), Pss 117, 91 (for the king, <locus from="20r" to="21v"/>).</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.3.4">
+                                        <locus from="21v" to="23v"/>
+                                        <title type="complete" xml:lang="gez">On how to sing the chants of the Old and New Testaments.</title>
+                                        <textLang mainLang="gez"/>
+                                        <note>The biblical chants are the following: Ex 12:1-18 (<locus from="21v" to="22r"/>), Deut 32:1-21 (preceeded by the word <foreign xml:lang="gez">ዘዳግም፡</foreign>, <locus target="#22r"/>), 
+                                            Deut 32:22-43 (preceeded by the word <foreign xml:lang="gez">ዘሣል<supplied reason="explanation">ስ</supplied>፡</foreign>, <locus target="#22r"/>),
+                                            1Kings 2:1-10, Is 38:10-20, prayer of Manasseh ( <locus target="#22v"/>), Jon 2:3-10, Dan 3:52-56, Hab 3:1-19, Is 26:9-20, Lk 1:46-55, Lk 1:68-79.
+                                            Then follow verses from the chants for the feast days of the year (<locus from="22v" to="23v"/>).</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.3.5">
+                                        <locus from="23v" to="26r"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT1113Anqasa"/>
+                                        <textLang mainLang="gez"/>
+                                        <note></note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.3.6">
+                                        <locus from="26r" to="27r"/>
+                                        <title type="complete" xml:lang="gez">Conclusion of the office for the night</title>
+                                        <textLang mainLang="gez"/>
+                                        <note>This section includes Lk 2:29-32, Dan 3:57-87, Pss 148-149, an admonition on the reading of the synaxarium and of the miracles, a supplication for the
+                                            peace.</note>
+                                    </msItem>         
+                                </msItem> 
+                                <msItem xml:id="ms_i1.4">
+                                    <locus from="27r" to="33v"/>
+                                    <title type="complete" xml:lang="gez" ref="LIT2301Sebhat">Fourth section, on the office for the morning.</title>
```

Comment:
Again, "office for the morning" could refer to many things, but "Sǝbḥata nagh" is precise and unambiguous.

I would suggest using LIT3186Meeraf#Sebhat. I wonder if there is a reason for the existence of LIT2301Sebhat except for the EAe index. It only seems to be attested in Meerafs, so maybe it can be deleted